testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/saml/CreateArtifactMessageStepBuilder.java

```diff
@@ -30,6 +39,7 @@
     private final SamlClientBuilder clientBuilder;
     private final String issuer;
     private String lastArtifact;
+    private ArtifactResolver artifactResolver = new DefaultSamlArtifactResolver();
```

Comment:
Does this take into account the SPI configuration? It seems hardcoded.

Reply:
Has this is our default implementation for Keycloak, so I would say it make sense to hard code it in our tests. No ?